<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AOChat</class>
 <widget class="QWidget" name="AOChat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat: (character)</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item row="4" column="0">
    <widget class="QStackedWidget" name="stacked_widget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_normal">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="holdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>76</width>
             <height>28</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
  border-image: url(:/base/themes/default/holdit.png);
}

:checked {
  border-image: url(:/base/themes/default/holdit_selected.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="objection">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>76</width>
             <height>28</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
  border-image: url(:/base/themes/default/objection.png);
}

:checked {
  border-image: url(:/base/themes/default/objection_selected.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="takethat">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>76</width>
             <height>28</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
  border-image: url(:/base/themes/default/takethat.png);
}

:checked {
  border-image: url(:/base/themes/default/takethat_selected.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="custom_interjection">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>76</width>
             <height>28</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
  border-image: url(:/base/themes/default/custom.png);
}
:disabled {
  border-image: none
}
:checked {
  border-image: url(:/base/themes/default/custom_selected.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Emotes</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QListWidget" name="emotes">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>52</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>140</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::TabFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QListWidget {
background: transparent;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="dragDropMode">
             <enum>QAbstractItemView::DragDrop</enum>
            </property>
            <property name="flow">
             <enum>QListView::LeftToRight</enum>
            </property>
            <property name="isWrapping" stdset="0">
             <bool>false</bool>
            </property>
            <property name="resizeMode">
             <enum>QListView::Fixed</enum>
            </property>
            <property name="viewMode">
             <enum>QListView::IconMode</enum>
            </property>
            <property name="itemAlignment">
             <set>Qt::AlignCenter</set>
            </property>
            <item>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normalon>../../base-full/characters/Apollo/emotions/button1_on.png</normalon>
              </iconset>
             </property>
            </item>
            <item>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normalon>../../base-full/characters/Apollo/emotions/button2_on.png</normalon>
              </iconset>
             </property>
            </item>
            <item>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normalon>../../base-full/characters/Apollo/emotions/button3_on.png</normalon>
              </iconset>
             </property>
            </item>
            <item>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normalon>../../base-full/characters/Apollo/emotions/button4_on.png</normalon>
              </iconset>
             </property>
            </item>
            <item>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normalon>../../base-full/characters/Apollo/emotions/button5_on.png</normalon>
              </iconset>
             </property>
            </item>
            <item>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normalon>../../base-full/characters/Apollo/emotions/button6_on.png</normalon>
              </iconset>
             </property>
            </item>
            <item>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normalon>../../base-full/characters/Apollo/emotions/button7_on.png</normalon>
              </iconset>
             </property>
            </item>
            <item>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normalon>../../base-full/characters/Apollo/emotions/button8_on.png</normalon>
              </iconset>
             </property>
            </item>
            <item>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normalon>../../base-full/characters/Apollo/emotions/button9_on.png</normalon>
              </iconset>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="gridWidget" native="true">
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="1" column="1">
           <widget class="QCheckBox" name="flip">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="focusPolicy">
             <enum>Qt::TabFocus</enum>
            </property>
            <property name="text">
             <string>Flip</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="preanim">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>2</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="focusPolicy">
             <enum>Qt::TabFocus</enum>
            </property>
            <property name="text">
             <string>Preanimation</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="5">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <widget class="QComboBox" name="emote">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>2</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="color">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="focusPolicy">
               <enum>Qt::TabFocus</enum>
              </property>
              <item>
               <property name="text">
                <string>White</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Green</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Red</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Orange</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Blue</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Yellow</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Rainbow</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Pink</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Cyan</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="side">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="focusPolicy">
               <enum>Qt::TabFocus</enum>
              </property>
              <item>
               <property name="text">
                <string>Witness</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Defense</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Prosecution</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Judge</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Helper Def.</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Helper Pro.</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Juror</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Seance</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="realization">
            <property name="focusPolicy">
             <enum>Qt::TabFocus</enum>
            </property>
            <property name="text">
             <string>Realization</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="no_interrupt">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>2</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="focusPolicy">
             <enum>Qt::TabFocus</enum>
            </property>
            <property name="text">
             <string>No interrupt</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QLineEdit" name="showname">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>24</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>Custom name</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_spectator">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Spectator Mode</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="AOChatEntry" name="chat_entry">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>48</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Message</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AOChatEntry</class>
   <extends>QTextEdit</extends>
   <header location="global">widgets/aochatentry.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>chat_entry</tabstop>
  <tabstop>holdit</tabstop>
  <tabstop>objection</tabstop>
  <tabstop>takethat</tabstop>
  <tabstop>custom_interjection</tabstop>
  <tabstop>emotes</tabstop>
  <tabstop>emote</tabstop>
  <tabstop>color</tabstop>
  <tabstop>side</tabstop>
  <tabstop>preanim</tabstop>
  <tabstop>flip</tabstop>
  <tabstop>realization</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
